Change example + Sorting by emdb as default + Add Nucleoprotein on search + Change filters

p3rcypj opened the pull request:
### :pushpin: References

-   **Issue:** Closes https://app.clickup.com/t/28a6nrt, https://app.clickup.com/t/28a6p5n, https://app.clickup.com/t/28a6p9c, https://app.clickup.com/t/29cdtau

### :memo: Implementation

Changed _'mouse'_ example to _' Llama'_. Results ordered descending by EMDB as default. Does not apply if the user begins to sort by another column. Added _'Nucleoprotein'_ when searching _'NTD'_ or _'CTD'_ on the proteome. Added updated data. Removed old json files and types. Addded new filters and validations; now: 'PDB-REDO', 'CSTF', 'Phenix'. Isolde and Refmac joined together. Now we are gathering validations by `refmodels[].source` property. Added validation method and source descriptions on hover each validation. Now we are getting the source by reference instead of nesting the properties inside each pdb.

### :art: Screenshots

![image](https://user-images.githubusercontent.com/43061485/167109571-bb3760e5-7b09-41cd-9895-c5d4cc1f3d82.png)
![image](https://user-images.githubusercontent.com/43061485/167109640-42ab3002-41f4-4df6-ba0d-977378eae7b9.png)
![image](https://user-images.githubusercontent.com/43061485/167109716-ecee6740-6529-43bf-b678-0afab95623a4.png)

Commit:
Weekly data update
- Ligands

Commit:
Weekly data update
- Structures

Commit:
Weekly data update
- Structures

Commit:
Merge remote-tracking branch 'eyeseetea/development-old-viewer' into feature/data-updates

Commit:
Weekly data update
- Refmac

Commit:
Weekly data update
- Refmac

p3rcypj committed:
Change example: mouse -> llama

p3rcypj committed:
Set sorting by emdb as default

p3rcypj committed:
Add Nucleoprotein along with its proteins on search

p3rcypj committed:
Selected entries typo

tokland commented on app/assets/javascripts/covid19/src/webapp/components/structures-table/StructuresTable.tsx:
```diff
@@ -31,6 +31,12 @@ export const StructuresTable: React.FC<StructuresTableProps> = React.memo(props
+    const [sortModel, setSortModel] = React.useState<GridSortModel>([
+        {
+            field: "emdb",
+            sort: "desc" as GridSortDirection,
```
[ultra-minor] No need for `as GridSortDirection` because of the `FieldStructure` hint at useState. No need to change.

tokland reviewed (approved):
Looking all good, code-wise!

Commit:
Weekly data update
- organisms

Commit:
Weekly data update
- Ligands

Commit:
Weekly data update
- Structures

Commit:
Weekly data update
- Organisms
- Ligands

Commit:
Weekly data update
- Structures

p3rcypj committed:
Don't show RefDoc ID in case it's void

p3rcypj committed:
Remove refEMDB and refPDB from Details (they are not being used)

p3rcypj committed:
Merge remote-tracking branch 'upstream/feature/data-updates' into feature/minor-changes-search

Commit:
Weekly data update
- Organisms
- Ligands

Commit:
Weekly data update
- Structures

p3rcypj committed:
Remove old json, RefEMDB RefPDB from types, change filters and vaidations infrastructure

p3rcypj committed:
Update i18n

tokland reviewed (commented):
Code looks very good, just a couple of changes and we're ready to go.

p3rcypj committed:
Add exact match search and remove console.log

p3rcypj committed:
Remove getValidationSource from repository and put into domain

p3rcypj committed:
Fix warnings

p3rcypj committed:
Remove link and fix tooltip on view more dialog

p3rcypj committed:
Fix warning and update i18n

tokland committed:
Order rows by score when using search

tokland reviewed (approved):
All good! :+1:

tokland committed:
Refactor search/sorting on structures table

Commit:
Weekly data update
- Refined Models Sources & Methods

Commit:
Weekly data update
- Structures

Commit:
Weekly data update
- Structures

Commit:
Weekly data update
- Ligands

Commit:
Weekly data update
- Ligands
Include new fields with InChIkey and PubChem Id

Commit:
Add new section to Ligand for ImageData

Just a simgle example (proposal) for Adenine
Not all the new fields to be displayed in the UI

Commit:
Add new section to Ligand for ImageData

Just a simgle example (proposal) for Adenine
Not all the new fields to be displayed in the UI

p3rcypj committed:
Merge remote-tracking branch 'upstream/feature/data-updates' into feature/minor-changes-search
